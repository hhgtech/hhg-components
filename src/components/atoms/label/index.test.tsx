import React from 'react'

import { render, cleanup } from '@testing-library/react'
import 'jest-styled-components'

import { Label } from '.'
import MockTheme from '../../../misc/mockTheme'

afterEach(cleanup)

test('it renders', () => {
  const label = render(
    <MockTheme>
      <Label>Hello World</Label>
    </MockTheme>,
  )

  const self = label.container.firstChild

  expect(self).toMatchSnapshot()

  expect(self).toHaveTextContent('Hello World')
})
