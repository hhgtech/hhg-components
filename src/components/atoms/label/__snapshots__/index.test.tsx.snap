// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
.c0 {
  font-size: 13px;
  color: #737373;
  line-height: 20px;
}

<label
  class="c0"
>
  Hello World
</label>
`;
